test/backend.js

```diff
@@ -1,29 +1,51 @@
 var Backend = require('../lib/backend');
 var expect = require('chai').expect;
 var sinon = require('sinon');
+var logger = require('../lib/logger');
 
 describe('Backend', function() {
   var backend;
-  var agent = {
-    custom: {
-      foo: 'bar'
-    }
-  };
-  var fetchOptions = {
-    snapshotOptions: {
-      fizz: 'buzz'
-    }
-  };
-
-  beforeEach(function() {
-    backend = new Backend();
```

Comment:
It feels a bit odd to remove the backend setup from `beforeEach` but keep the tear-down in `afterEach` 🤔 

I suppose it's fair to expect that every test case here will construct a Backend somehow, since this is a test for backend.js. And this way tests won't forget to close their backend.

Reply:
I think it should be fine; if tests don't set up a `backend`, then the test will throw when trying to tear it down. We can rejig the tests again in that case.